### DISCUSSION TITLE
I have column-margin figure that render fine in HTML but not so much when output is pdf

### DISCUSSION THREAD

**Discussion by IceManTBM3** [quarto-dev/quarto-cli#9620] [Q&A]
I have content throughout our site that has column-margin images. When I do a browser print to pdf fille, they print in the main body, but using this code below they just get truncated as do some tables. Is there a way to get it all in one clean main column?

Thanks in advance

````qmd
---
title: RMBS Product Tear Sheet
author: "ICE Sustainable Finance"
nocite: |
  @*
format:
  pdf:
    header-includes: |
      \titlehead{\includegraphics[width=6in]{img/ICE77x76.png}}
    toc: true
    toc-depth: 2
    number-sections: true
    number-depth: 2
    output-file: "RMBS-Tear"
    output-ext:  "pdf"
    cover-image: img/ICE77x76.png
    css: styles.css
    hyperrefoptions: draft
    colorlinks: true
    tbl-colwidths: false
    df-print: kable
    fig-align: left
    fig-pos: 'h'
    include-in-header: 
       - "styles.tex"
editor: source
---




```{r}
#| echo: false
#| eval: true
process_qmd <- function(file, fpath_in = "/", fpath_out = "/") {
  doc <- readLines(file)
  end_yaml <- which(doc == "---")[2]
  out_doc <- doc[seq(end_yaml+1, length(doc))]
  if (fpath_in != fpath_out) {
    out_doc <- stringr::str_replace_all(out_doc, fpath_in, fpath_out)
  }
  res <- knitr::knit_child(text = out_doc, quiet = TRUE, options = list(eval = FALSE, echo = TRUE))
  return(res)
}
```

# Mission/Overview

```{r}
#| output: asis
#| echo: false
#| eval: true
#| message: false
idx <- process_qmd("index.qmd")
cat(unlist(idx), sep = '\n')
```

# Disclaimer
```{r}
#| output: asis
#| echo: false
#| eval: true
#| message: false
idx <- process_qmd("disclaimer.qmd")
cat(unlist(idx), sep = '\n')
```
````

![image](https://github.com/quarto-dev/quarto-cli/assets/129208946/23c5ab6b-73eb-43dc-a98b-bce092ac14cb)

**Comment by mcanouil** [comment]
Hi @IceManTBM3! When sharing code, please make sure to format it properly. It is nearly impossible for others to use unformatted code as the one you shared. Also, note that an example is provided when you open a discussion. We appreciate your contributions, and if you need any assistance with formatting, feel free to ask. 😊

We would really appreciate if you could take some time and account for our formatting requests as it's not your first time and not the first formatting request you receive.
Without a well-formatted example, it becomes challenging for us to provide helpful feedback or replies, and impossible for future users to use your discussion. Thank you for your understanding and cooperation!

<img width="689" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/9f5635c1-ab92-47e0-b885-84abbab1f9d6">

**Reply by mcanouil** [reply]
Additionally, we kindly ask for a **small self-contained "working" (reproducible)** example, such as a complete Quarto document or a Git repository. Unfortunately, your current code doesn't meet these criteria, which makes it challenging for us to provide effective assistance. Thank you for understanding!

**Comment by IceManTBM3** [comment]
Ok I think I updated the code Ok now

**Reply by mcanouil** [reply]
See https://github.com/quarto-dev/quarto-cli/discussions/9620#discussioncomment-9358869

Unfortunately, your code is not valid!

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by IceManTBM3** [reply]
Ok, think I did that... GTK RE ticks
